test/e2e/operator.go

```diff
@@ -132,6 +144,11 @@ var _ = Describe("ARO Operator - Geneva Logging", func() {
 
 		err := wait.PollImmediate(30*time.Second, 15*time.Minute, mdsdReady)
 		Expect(err).NotTo(HaveOccurred())
+		if err != nil {
```

Comment:
This doesn't work.

Reply:
Why this does not work? We get timeout errors in the tests where `mdsdReady` fails(?)